src/libraries/System.Text.Json/src/System/Text/Json/Serialization/ReferenceEqualsEqualityComparer.cs

```diff
@@ -21,7 +22,7 @@ bool IEqualityComparer<T>.Equals(T x, T y)
 
         int IEqualityComparer<T>.GetHashCode(T obj)
         {
-            return obj!.GetHashCode();
+            return RuntimeHelpers.GetHashCode(obj);
```

Comment:
```suggestion
            return RuntimeHelpers.GetHashCode(obj!);
```

CI is failing because of this.

Reply:
Yes, thanks. Just found this error

Reply:
I see; you applied the suggestion to the file `src/libraries/Common/src/System/Collections/Generic/ReferenceEqualityComparer.cs`

This one still needs to be changed. I will commit it and expect for the build to pass. 
My bad, I should've noticed this on the review.